website/src/pages/playground/index.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect, useState, useReducer } from 'react';
+import classnames from 'classnames';
+import MonacoEditor from 'react-monaco-editor';
+import Layout from '@theme/Layout';
+import useTheme from './useTheme';
+import Worker from 'worker-loader!./worker.js';
+import styles from './styles.module.css';
+import RunIcon from './runIcon';
+import useWindowSize from './useWindowSize';
+
+const vsDarkTheme = {
+  base: 'vs-dark',
+  inherit: true,
+  rules: [{ background: '121212' }],
+  colors: {
+    'editor.background': '#121212',
+  },
+};
+
+const worker = new Worker();
+
+function reducer(state, action) {
+  switch (action.type) {
+    case 'request':
+      return { ...state, loading: true, lastTime: new Date() };
+    case 'log':
+      return {
+        ...state,
+        output: state.output + action.log,
+      };
+    case 'success':
+      return {
+        ...state,
+        loading: false,
+        output: action.result,
+        ellapsed: new Date() - state.lastTime + ' ms',
+      };
+  }
+}
+
+const initialState = {
+  loading: false,
+  output: '',
+  lastTime: undefined,
+  ellapsed: undefined,
+};
+
+function Playground() {
+  const [{ loading, ellapsed, output }, dispatch] = useReducer(
+    reducer,
+    initialState
+  );
+  const [input, setInput] = useState('const a = 1; \nprint(a);');
+  const [args, setArgs] = useState('-help');
```

Comment:
Yes sure, I was also thinking about these " cli shortcuts". Do we want other buttons for options we use a lot, like `[?] [Bytecode] [AST]` ?

Yes totally, I can also remove the "log" thing in this case.

On a side note, what I "saw" ultimately was 
- left: args input + help + shortcuts + editor
- right: different "viewers" (JSON tree, bytecode, plain text) + buttons to switch  + interactions between the editor and the viewer, like 
  - fold JSON node <-> fold code
  - highlight statement <-> highlight lexer loc (AST / byte code) etc but it requires lot more work :sweat_smile:

It can also be incrementally added in the future

Reply:
The original explorer was warming Hermes up immediately on page load, so usually by the time the user started doing something it was already warmed up. Are we doing the same here?

Reply:
@exced that plan sounds good to me as a followup!

Reply:
@tmikov We're still warming it up upfront